GameBoard/Views/Shared/Components/GameEventsList/GameEventsList.cshtml

```diff
@@ -2,33 +2,40 @@
 
 <h1 class="mb-4 mr-xl-5 pr-xl-5">Your game events</h1>
 <div id="game-events-container">
-    <table id="game-events-table" class="table table-striped table-dark">
-        <thead>
-        <tr>
-            <th>Name</th>
-            <th>Creator</th>
-            <th>Status</th>
-        </tr>
-        </thead>
-        <tbody>
-        @foreach (var gameEvent in Model.Created.Concat(Model.Accepted))
-        {
-            <tr onclick="window.location.href = '/GameEvent/GameEvent/@gameEvent.Id'">
-                <td>@gameEvent.Name</td>
-                <td>@await Component.InvokeAsync("UserCard", new {user = gameEvent.Creator})</td>
-                <td class="text-success">Active</td>
+    @if (!Model.Created.Any() && !Model.Created.Any() && !Model.Created.Any())
```

Comment:
You meant !Model.Created.Any() && !Model.Accepted.Any() && !Model.Pending.Any(), didn't you?

Reply:
Copy paste development shame. Fixed.